src/piadm.sh

```diff
@@ -293,6 +296,7 @@ install() {
 		# Explicit boot stamp or URL.
 
 		# Do a URL reality check.
+		vecho "Attempting download of $1"
 		"${CURL[@]}" -o "${tdir}/download" "$1"
```

Comment:
Same with `--progress-bar` here

Reply:
Let me see what I can do.  It's easy enough to test...

Reply:
... seems to work. Please look over the new diff, @bahamat !